core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1749,7 +1749,7 @@ protected static function setCurrentRecordCache(string|int $id, string $table, a
 	 * @throws AccessDeniedException     if the current user has no read permission
 	 * @return array<string, mixed>|null
 	 */
-	public function getCurrentRecord(string|int $id = null, string $table = null): ?array
+	public function getCurrentRecord(string|int $id = null, string $table = null, bool $throw = true): ?array
```

Comment:
I don‘t like the API. IMHO the method should always throw the exception and we should use a try-catch block if we want to handle the case. So instead of

```php
$currentRecord = $this->getCurrentRecord($row['id'], $v, false);

if ($currentRecord === null)
{
	continue;
}
```

it should be

```php
try
{
	$currentRecord = $this->getCurrentRecord($row['id'], $v);
}
catch (AccessDeniedException $e)
{
	continue;
}
```